simple_rpc/simple_rpc.py

```diff
@@ -120,22 +143,17 @@ def open(self):
 
     def close(self):
```

Comment:
This function is only called once by the destructor.
Alternatively, it is called explicitly by the user.

Reply:
ok